src/CSharp/EasyMicroservices.Serialization.Tests/EasyMicroservices.Serialization.Tests.csproj

```diff
@@ -9,6 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="3.2.0">
```

Comment:
Use EasyMicroservices.Tests packages it has all of them.

Reply:
coverlet.collector  + coverlet.msbuid
does not exist in EasyMicroservices.Tests
probably because it's used with PrivateAssets

![Capture](https://user-images.githubusercontent.com/63418929/210184000-fa3e5e12-6760-47ea-8895-be43a8641326.PNG)

Reply:
@Azaferany I published a new version of EasyMicroservices.Tests Packages and removed PrivateAssets.